ironfist/src/cpp/adventure/adv.cpp

```diff
@@ -137,4 +137,16 @@ void advManager::DoEvent(class mapCell *cell, int locX, int locY) {
 		ScriptSignal(SCRIPT_EVT_VISIT_CAMPFIRE, tmp.str());
 	}
 	this->DoEvent_orig(cell, locX, locY);
+}
+
+int advManager::MapPutArmy(int x, int y, int monIdx, int monQty) {
+	int cellIdx = x * gpGame->map.height + y;
+	gpGame->map.tiles[cellIdx].objectIndex = monIdx;
+	gpGame->map.tiles[cellIdx].extraInfo = monQty;
+	gpGame->map.tiles[cellIdx].objTileset = 12;
```

Comment:
Oh. Looks like this is actually in EDITOR2W.h . This one is OBJ_TILESET. Go ahead and put that enum somewhere.